core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250912_workflow_actual_cost.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.31.xsd">
+    <changeSet logicalFilePath="dummy" author="davidan" id="workflow_actual_cost">
+        <!-- create WORKFLOW_COST table -->
+        <createTable tableName="WORKFLOW_ACTUAL_COST">
+            <column name="WORKFLOW_ID" type="bigint(19)">
+                <constraints nullable="false"
+                             unique="true"
+                             referencedTableName="WORKFLOW"
+                             referencedColumnNames="ID"
+                             foreignKeyName="FK_WORKFLOW_ACTUAL_COST_WORKFLOW"
+                             deleteCascade="true"/>
+            </column>
```

Comment:
with deleteCascade="true" on the foreign key here, rows in WORKFLOW_ACTUAL_COST will automatically be deleted when a workspace is deleted (because deleting a workspace deletes its workflows)